jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/LambdaHandlerResolver.kt

```diff
@@ -67,5 +68,11 @@ interface LambdaHandlerResolver {
      */
     fun shouldShowLineMarker(handler: String): Boolean = false
 
+    /**
+     * Given a handler and the element it originated from, produce the appropriate Action that can
+     * be used to create a Lambda Function.
+     */
+    fun generateCreateLambdaFunctionAction(handlerName: String, element: PsiElement): CreateLambdaFunction
```

Comment:
I'm not sure this belongs here, the create action should be the same for all languages right? It's only the packaging that differs.